docs/content/preview/launch-and-manage/monitor-and-alert/metrics/throughput.md

```diff
@@ -63,6 +97,20 @@ The YSQL throughput can be viewed as an aggregate across the whole cluster, per
 | `handler_latency_yb_ysqlserver_SQLProcessor_OtherStmts` | microseconds | counter | The time in microseconds to parse and execute all other statements apart from the preceding ones listed in this table. This includes statements like PREPARE, RELEASE SAVEPOINT, and so on. |
 | `handler_latency_yb_ysqlserver_SQLProcessor_Transactions` | microseconds | counter | The time in microseconds to execute any of the statements in this table.| -->
 
+## YCQL query processing
+
+YCQL query processing metrics represent the total inclusive time it takes YugabyteDB to process a YCQL statement after the query processing layer begins execution. These metrics include the time taken to parse and execute the YCQL statement, replicate over the network, the time spent in the storage layer, and so on. The preceding metrics do not capture the time to deserialize the network bytes and parse the query.
```

Comment:
replicate over the network
-->
replicate over the network (in case of write operations)